tasks/copy_files_to_archive/test/test_copy_files_to_archive.py

```diff
@@ -226,9 +232,12 @@ def test_handler_one_file_fail_3x(self):
                                                   exp_upd_result])
         mock_ssm_get_parameter(5)
         try:
+            seconds = time.time()
```

Comment:
I think this will work. If you're just testing that some number of retries have been hit, another option is to `mock` a function that's called in each retry and [assert_has_calls](https://docs.python.org/3.3/glossary.html#term-mapping).

I like that method because it doesn't require you keep a timer/coordinate with sleep time. I'm fine if you want to stick with your method, I just wanted to offer another alternative.

Reply:
I'm actually trying to make sure the function used the sleep value I gave it. I should probably add a timer on the successful tests to cover my fix on excessive sleeps.

Reply:
Oof, forgot I could do that. I've mocked out 'sleep' calls in a few functions, and caught some test errors along the way.